parachain/pallets/erc20-app/src/lib.rs

```diff
@@ -88,6 +101,15 @@ decl_module! {
 			}
 
 			<asset::Module<T>>::do_burn(asset_id, &who, amount)?;
+
+			let message = Message {
+				token: asset_id,
+				sender: who.clone(),
+				recipient: recipient,
+				amount: amount
+			};
+			T::Commitments::add(Self::address(), message.encode());
+
 			Self::deposit_event(RawEvent::Transfer(asset_id, who.clone(), recipient, amount));
```

Comment:
Do we still need this?

Reply:
Yeah, we need to keep it until we can transition the relayer to use the new message/event RPC. Am keen to tackle that in a subsequent PR.